qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -367,9 +367,55 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def particle_number(n_orbitals, mapping="jordan_wigner"):
+    r"""Computes the particle number operator :math:`\hat{N}=\sum_\alpha \hat{n}_\alpha` in
+    the Pauli basis. Its eigenvalue is the number of particles in a given state of the
+    many-body system.
+
+    The particle number operator reads,
+
+    .. math::
+
+        \hat{N} = \sum_\alpha \hat{c}_\alpha^\dagger \hat{c}_\alpha.
+
+    The index :math:`\alpha` runs over the basis of single-particle orbitals
+    :math:`\vert \alpha \rangle` and :math:`\hat{c}^\dagger` and :math:`\hat{c}` are
+    the particle creation and annihilation operator, respectively.
+
+    Args:
+        n_orbitals (int): Number of orbitals. If an active space is defined 'n_orbitals'
+            is the number of active orbitals.
+        mapping (str): Specifies the transformation to map the fermionic operator to the
+            Pauli basis. Input values can be ``'jordan_wigner'`` or ``'bravyi_kitaev'``.
+
+    Returns:
+        pennylane.Hamiltonian: the fermionic-to-qubit transformed observable
+
+    **Example**
+
+    >>> n_orbitals = 2
+    >>> pn_obs = particle_number(n_orbitals, mapping="jordan_wigner")
+    >>> print(pn_obs)
```

Comment:
```suggestion
    >>> n = particle_number(n_orbitals, mapping="jordan_wigner")
    >>> print(n)
```
Since we usually use N to denote this operator

Reply:
Nice, can we use ``N``?